src/main/java/net/commments/object/CommentTable.java

```diff
@@ -4,6 +4,12 @@
 
     void checkCommentInTable(int numberComment);
```

Comment:
```java
void checkCommentInTable(int numberComment);
boolean isCommentTextCorrect(String name);
boolean isCommentNumberCorrect(String number);
boolean isCommentCategoryCorrect(String category);
 ```
All these methods perform one task: check comment's existence on the page by criteria. Current methods names don't provide an unambiguous definition of what they do. Please try to remave all of them with a common naming strategy.

Reply:
fixed